addons/fire/XEH_postInit.sqf

```diff
@@ -13,6 +13,19 @@
     TRACE_1("settingsInit", GVAR(enabled));
     if (!GVAR(enabled)) exitWith {};
 
+    ["AllVehicles", "Killed", {
+        params ["_vehicle", "_killer"];
+
+        if !(local _vehicle) exitWith {};
+
+        if (getText (configOf _vehicle >> "destrType") == "") exitWith {};
+
+        private _radius = 1.5 * ((boundingBoxReal _vehicle) select 2);
```

Comment:
Radius should probably be the smallest axis in the xy-plane * 1.5 for gameplay reasons

Reply:
I'm worried about that implementation for things like the HEMTT. One side is much longer to the point where touching the back of the vehicle might not burn the unit due to the distance from the model center.

Reply:
I don't know, I thought it may look goofy if you aren't near the fire and you still burn. Worst case we change it later